dicom-mr-classifier.py

```diff
@@ -597,7 +587,25 @@ def dicom_classify(zip_file_path, outbase, timezone, config=None):
                 classification = {'Custom': ['N/A']}
         elif dcm.get("Modality") != "MR":
             classification = {}
-        dicom_file["classification"] = classification
+        dicom_file["classification"].update(classification)
+
+    # If no pixel data present, make classification intent "Non-Image"
+    if not hasattr(dcm, "PixelData"):
+        nonimage_intent = "Non-Image"
+        # If classification is a dict, update dict with intent
+        if isinstance(dicom_file["classification"], dict):
+            classification  = dicom_file['classification']
+            # If intent not present, add it with nonimage_intent
+            if 'Intent' not in classification:
+                classification['Intent'] = nonimage_intent
```

Comment:
I think this works, but @AndysWorth does this affect BIDS?

Reply:
It depends on the template being used.  Looking at this [BIDS Template Comparison](https://docs.google.com/spreadsheets/d/1u3wHsQRzu80DDLGzg5_QxJo5s3-iikDBVpdNmc_tqLc/edit#gid=36924381), it probably will not have an effect if I understand what is going on here because the intents some templates care about is if it is structural or functional.  This should be fine especially since we're switching to the ReproIn template (see the "Recommended" tab) which does not use Classification (as much).